androidsdk/src/main/java/com/wootric/androidsdk/SurveyManager.java

```diff
@@ -348,6 +349,7 @@ private void showSurveyFragment() {
             }
         } catch (NullPointerException e) {
```

Comment:
Oh boy, didn't see that one earlier, very naughty to catch a `NullPointerException`. Not something to change for now, but as an improvement for later on it could be good to check if variables are null / not null before using them with the risk of raising an exception.